great_expectations/render/renderer/content_block/content_block.py

```diff
@@ -377,12 +417,34 @@ def _process_content_block(
             content_block.header = header
 
     @classmethod
-    def _get_content_block_fn(cls, expectation_type):
+    def _get_content_block_fn(
+        cls,
+        expectation_type: str,
+        expectation_config: ExpectationConfiguration | None = None,
+    ) -> Callable | None:
+        # Prioritize `description` param on Expectation before falling back to renderer
+        content_block_fn = cls._get_content_block_fn_from_expectation_description(
+            expectation_config=expectation_config,
+        )
+        if content_block_fn:
+            return content_block_fn
+
         content_block_fn = get_renderer_impl(
             object_name=expectation_type, renderer_type=LegacyRendererType.PRESCRIPTIVE
         )
         return content_block_fn[1] if content_block_fn else None
 
+    @classmethod
+    def _get_content_block_fn_from_expectation_description(
+        cls, expectation_config: ExpectationConfiguration | None
+    ) -> Callable | None:
+        if expectation_config:
+            expectation = expectation_config.to_domain_obj()
+            description = expectation.description
+            if description:
+                return cls._render_expectation_description
+        return None
```

Comment:
Since `description` is a `ClassVar`, we need the `Expectation` to retrieve it - we want the `Expectation` to be the source of truth moving forward I just don't know if/when we'll have the bandwidth to remove config